engine/crates/engine-v2/schema/src/lib.rs

```diff
@@ -268,12 +269,30 @@ impl From<InputObjectId> for Definition {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone, Copy)]
 pub struct Type {
     pub inner: Definition,
     pub wrapping: Wrapping,
 }
 
+impl Type {
+    /// Determines whether a varia
```

Comment:
This comment needs finished.  a link to the list coercion bit of the spec would probably be good to include if you weren't already going to.